common/setups/rasr/gmm_system.py

```diff
@@ -1007,6 +1074,63 @@ def run_vtln_sat_step(self, step_args):
                     **step_args.sdm_args,
                 )
 
+    def run_recognition_step(
```

Comment:
Can we maybe expand this step to incorporate also sat_recognition?

E.g. have a flag in the step_args  and then do smth like
```
recog_funktion = self.sat_recognition if step_args["sat"] else self.recognition
```

This way we could simplify / deduplify some code above in the `run_sat_step`